sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/jackson/JacksonAdapter.java

```diff
@@ -117,12 +117,14 @@ public String serialize(Object object, SerializerEncoding encoding) throws IOExc
 
     @Override
     public String serializeRaw(Object object) {
+        final ClientLogger logger = new ClientLogger(JacksonAdapter.class);
```

Comment:
Why are we creating a new one here instead of having a private final one in the class?

Reply:
Removed